package.json

```diff
@@ -163,7 +163,7 @@
     "cross-env": "^7.0.2",
     "cypress": "^12.14.0",
     "eslint": "8.34.0",
-    "eslint-config-airbnb": "19.0.4",
```

Comment:
updating package versions should happen in a separate PR, please revert this line change